tests/test-summary-view.sh

```diff
@@ -49,18 +49,18 @@ ${CMD_PREFIX} ostree --repo=repo pull --mirror origin
 assert_has_file repo/summary
 output=$(${OSTREE} summary --view)
 echo "$output" | sed -e 's/^/# /'
-echo "$output" | grep --quiet --no-messages "* main"
-echo "$output" | grep --quiet --no-messages "* other"
-echo "$output" | grep --quiet --no-messages "ostree.summary.last-modified"
-echo "$output" | grep --quiet --no-messages "Static Deltas (ostree.static-deltas): {}"
+echo "$output" | grep --quiet --no-messages --fixed-strings "* main"
+echo "$output" | grep --quiet --no-messages --fixed-strings "* other"
+echo "$output" | grep --quiet --no-messages --fixed-strings "ostree.summary.last-modified"
+echo "$output" | grep --quiet --no-messages --fixed-strings "Static Deltas (ostree.static-deltas): {}"
```

Comment:
Among other nice things, `libtest-core.sh`'s `assert_file_has_content` will print the non-matching value on failure which is what one wants for CI tests.